docs/using/custom-images.md

```diff
@@ -0,0 +1,74 @@
+# Building a custom set of images
+
+This section describes how to build a custom set of images.
+It may be helpful if you need to change Ubuntu or Python version, or make a significant change to the build process itself.
+
+## Building stack images with custom arguments
+
+A selection of prebuilt images are available from Quay.io,
+however, it's impossible to cater to everybody's needs.
+For extensive customization with an automated build pipeline,
+you may wish to create a [community-maintained stack](https://jupyter-docker-stacks.readthedocs.io/en/latest/contributing/stacks.html),
```

Comment:
```suggestion
you may wish to create a [community-maintained stack](../contributing/stacks),
```
I'm not sure if this is the correct format for the internal link

Reply:
It is, thanks!
I simply copied this section from the other file, but it's much better to use internal links